book/website/community-handbook/content.md

```diff
@@ -0,0 +1,139 @@
+(ch-content)=
+# Writing Content
+
+As _The Turing Way_ project grows and evolves, we actively welcome and encourage continuous contributions from community members in the form of new chapters and case studies. 
+However, these new contributions need to be consistent with the overall theme, purpose, format, and style of the book's existing content.
+
+Since _The Turing Way_ is written asynchronously by multiple authors around the world, we have created several templates to guide future content contributions in order to preserve the book's consistency as it grows.
+
+This chapter provides a brief overview of these templates. 
+You can find each one in the `template` [directory](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/templates) of the _The Turing Way's_ Github repository and we encourage you to use them if you wish to contribute to content to _The Turing Way_. 
+Also, ensure that you read our [Contributor Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) in addition to our [style](https://the-turing-way.netlify.app/community-handbook/style.html) and [consistency](https://the-turing-way.netlify.app/community-handbook/consistency.html) recommendations before you write.
+
+(ch-content-structure)=
+## Book Structure
```

Comment:
Depending on how the chapter is reorganized, this paragraph might go above a bit (or it might stay here). We have to see.